doc/classes/ResourceImporterWAV.xml

```diff
@@ -23,11 +23,12 @@
 			The end loop point to use when [member edit/loop_mode] is [b]Forward[/b], [b]Ping-Pong[/b], or [b]Backward[/b]. This is set in samples after the beginning of the audio file. A value of [code]-1[/code] uses the end of the audio file as the end loop point.
 		</member>
 		<member name="edit/loop_mode" type="int" setter="" getter="" default="0">
-			Controls how audio should loop. This is automatically read from the WAV metadata on import.
-			[b]Disabled:[/b] Don't loop audio, even if metadata indicates the file should be played back looping.
-			[b]Forward:[/b] Standard audio looping.
-			[b]Ping-Pong:[/b] Play audio forward until it's done playing, then play it backward and repeat. This is similar to mirrored texture repeat, but for audio.
-			[b]Backward:[/b] Play audio in reverse and loop back to the end when done playing.
+			Controls how audio should loop. 
+			- [b]Detect From WAV:[/b] Uses loop information from the WAV metadata. 
+			- [b]Disabled:[/b] Don't loop audio, even if metadata indicates the file playback should loop.
```

Comment:
The prior sentence does insert the article so this is more so for consistency. Should it be "the" or "its"? Unsure right now, probably doesn't matter much, since the context is reasonably obvious.
```suggestion
			- [b]Disabled:[/b] Don't loop audio, even if the metadata indicates the file playback should loop.
```